src/app/modules/wallet/components/account-actions/account-actions.component.html

```diff
@@ -21,8 +21,18 @@
       <mat-icon>exit_to_app</mat-icon>
     </button>
   </a>
+  <a class="ml-3"
+    [routerLink]="[ '/dashboard', 'wallet','accounts','backup']">
+    <button mat-raised-button
+      color="accent"
+      class="large-btn ml-1">
+      Back up accounts
```

Comment:
```suggestion
      Back Up Accounts
```
For consistency with other buttons.